scripts/src/primazactl/utils/settings.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+import sys
+from primazactl.utils import logger
+
+dry_run = False
+output_yaml = False
```

Comment:
I'd propose to have an `output` variable that can be `None` or `yaml` as of now. In future we may want to add `json` or other formatters.

Reply:
I changed to be more to your liking. would be a minor update for a adding a new type either way.  Adding json would be about a 5 line change to the code, but a bit more for adding a test.